content/doc/book/using/remote-access-api.adoc

```diff
@@ -204,7 +204,7 @@ variable)
 * Folder support
 * Jobs API (build, buildInfo, buildWithParameters, config, create,
 delete, description, disable, enable, jobInfo, lastBuildNumber,
-lastBuidTimestamp and progressiveText)
+lastBuildTimestamp and progressiveText)
```

Comment:
`buildTimestamp`, actually.

Reply:
so no last?

Reply:
There are more things wrong with this content, so out of scope for this PR.